geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/Part.java

```diff
@@ -175,6 +175,34 @@ public String getString() {
     return CacheServerHelper.fromUTF((byte[]) this.part);
   }
 
+  @MakeNotStatic
```

Comment:
Strikes me as totally acceptable to be static. Even if we have multiple caches they could share the same map here.

Reply:
I changed the annotation to include: "not tied to the cache lifecycle".